pkg/api/v1/service/util_test.go

```diff
@@ -214,3 +214,30 @@ func TestNeedsHealthCheck(t *testing.T) {
 		},
 	})
 }
+
+func TestRequestsOnlyLocalTrafficForInternal(t *testing.T) {
+	checkRequestsOnlyLocalTrafficForInternal := func(expected bool, service *v1.Service) {
```

Comment:
Nit: I think I'd find this easier to understand if it was using the test tables approach that's fairly common in k/k.

Reply:
That makes sense to me. The rest of the tests in this file are all in this format though..